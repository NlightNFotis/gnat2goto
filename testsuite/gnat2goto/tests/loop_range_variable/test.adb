procedure Test is
   type Unsigned_8 is mod 2 ** 4;
   subtype s10_index is Unsigned_8 range 1 .. 10;
   length : Unsigned_8 := 4;
   j : s10_index;
begin
   j := 1;
   for k in s10_index range 1 .. length loop
      j := j + 1;
   end loop;
   pragma Assert (j < 4);

end Test;
