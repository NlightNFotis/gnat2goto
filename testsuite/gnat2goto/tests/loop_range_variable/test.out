[assertion.1] file test.adb line 9 Range Check: SUCCESS
[test.assertion.1] line 11 assertion j < 4: FAILURE
VERIFICATION FAILED
